### ISSUE TITLE
A JavaScript error occurred in the main process

### ISSUE THREAD

**Issue by cashkows-johan** [joeferraro/MavensMate#996]
* **MavensMate Desktop Version**: 11.beta.7

* **Editor**: Sublime

* **MavensMate Plugin Version**:

* **Platform/Version**: Windows 10 64-bit

* **Proxy Details**: None

* **Salesforce API Version**: 40

* **Salesforce Server**: cs84

Creating a new mavensmate project from the sublime interface.
Right click, Create mavensmate project, login and then click create project starts and then this error appears 
![image](https://user-images.githubusercontent.com/13367234/28367624-f0d95d14-6c91-11e7-8dc8-f009a6f410fc.png)

**Comment by joebauer-bib** [comment]
I'm experiencing the same issue. Have you found a solution?

**Comment by cashkows-johan** [comment]
Yes I removed all the xml tags in my package.xml except for the apex one and updated the index again and the project and the error has not happened again.

**Comment by joebauer-bib** [comment]
@cashkows-johan thank you!

**Comment by aonyeagwa** [comment]
Actually, I had Mavensmate Directories, and I deleted them and went back to install Mavensmate, and it allowed me to connect to projects, again. BOOM! 

Cheers,
AO
